### PR TITLE
Feature: Final step

### PR DESCRIPTION
**Pull request by NickEmma**
# Hi there 👋🏻

**Thanks for taking the time to review the project. Here are some things important to note about it:**

The path for the greeting in the front end is /greeting
The path for the greeting JSON in the rails API is greetings

## Now into the project...

- ✅ Create a greetings table with some greetings
- ✅ Define cross-origin to admit localhost frontend
- ✅ Define API endpoint to return a random greeting
- ✅ A link to the frontend PR can be found [here](https://github.com/NickEmma/hello-react-frontend)

# Final notes 📔

_Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Project Name
```
Comment (brytebee) [inline README.md:1]: - [ ] **Required:** Kindly switch the **Project Name** placeholder with the name of the projects, both back and frond end README's.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @NickEmma👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (NickEmma) [reply README.md:1]: @brytebee Thank you for your review

**Review by mikethreels — APPROVED** [review]
Hi Techie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**